core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -278,7 +283,7 @@
     }
   }
 
-  @media only screen and (min-width: 601px) and (max-width: 1500px) {
+  @media only screen and (min-width: 701px) and (max-width: 1500px) {
```

Comment:
Hm typically we shouldn't need both min and max width as long as the media queries are ordered correctly. Since we're developing desktop first, we should order by descending order of max width to be consistent.

Reply:
Fixed